### ISSUE TITLE
Sony 2 factor auth is broken again

### ISSUE THREAD

**Issue by GreatDaemon** [Bungie-net/api#1242]
#897 is happening now again for me for the past week following enabling 2fa on my Sony account.
It's the same issue and same errors as noted previously.
Can I help fix it?

**Comment by jshaffstall-bng** [comment]
@GreatDaemon Sorry for the delay! Are you still able to reproduce this issue?

PlayStation sign-in with 2-factor-auth seems to be working for us at the moment.

**Comment by GreatDaemon** [comment]
Hey @jshaffstall-bng . Thanks for looking into this. Right now it doesn't. I started using Steam to auth (cross save is awesome). I tried PS auth a week ago and had it happen again.
But - I checked with 2FA disabled on PS side. It may be an issue with Sony's auth where when 2fa is enabled, it doesn't progress the process but sends a prohibited response. My gut feeling is that Sony is releasing internal updates and whenever they do, 2fa auth tokens break a bit for a few days/weeks.
